src/examples/component-definition/json/ocpv4-example-component-definition.json

```diff
@@ -0,0 +1,209 @@
+{
+  "component-definition": {
+    "uuid": "a7ba800c-a432-44cd-9075-0862cd66da6b",
+    "metadata": {
+      "title": "OpenShift Container Platform v4 Example Component Definition",
+      "last-modified": "2001-12-09T14:00:00Z",
+      "version": "20220201",
+      "oscal-version": "1.2.0-alpha",
+      "roles": [
+        {
+          "id": "provider",
+          "title": "Provider"
+        },
+        {
+          "id": "organization-devsecops-engineer",
+          "title": "Organization DevSecOps Engineer"
+        }
+      ],
+      "parties": [
+        {
+          "uuid": "12bde908-adba-4a59-a6a3-198163f62a48",
+          "type": "organization",
+          "name": "RedHat",
+          "links": [
+            {
+              "href": "https://docs.openshift.com/",
+              "rel": "website"
+            }
+          ]
+        }
+      ]
+    },
+    "rules": [
+      {
+        "name": "OpenShift Container Platform v4 Kubelet Eviction Threshhold Settings",
+        "uuid": "a969feae-ae93-422f-8a95-9362adef826b",
+        "title": "Check hardening guidelines for OCP kubelet instances with OpenSCAP",
+        "condition": "kubelet is hardened with CIS OCPv4 baseline validated with OpenSCAP",
+        "condition-tests": [
```

Comment:
I am not sure of the purpose of condition (it is just text)? What does condition test and steps represent.

Reply:
For now? Just text. At this stage in OSCAL, it will be a short summary statement in the first version. In a later revision and with certain OSCAL internals adapted or enhanced, we can have a structured condition, just not at this time.